docs/source/api/user-guide/measurements/thresholded-noise-mechanisms.ipynb

```diff
@@ -0,0 +1,514 @@
+    "    dp.map_domain(dp.atom_domain(T=str), dp.atom_domain(T=float, nan=False)),\n",
+    "    # NOTE: L1 is changed to L2 in the input metric\n",
+    "    dp.l_02I(dp.absolute_distance(T=float)),\n",
+    "    scale=1.,\n",
+    "    threshold=20.0\n",
+    ")\n",
+    "\n",
+    "# invoke the measurement on some aggregate hashmap, to sample Gaussian(x, 1.) noise\n",
+    "print(\"noisy aggregate:\", m_gauss(aggregated))\n",
+    "\n",
+    "# we must know the sensitivity of `aggregated` to determine privacy params\n",
+    "#  3 kinds: Δ_0, Δ_1, Δ_∞\n",
+    "sensitivity = 1, 1.0, 1.0\n",
+    "print(\"(ρ, δ):\", m_gauss.map(d_in=sensitivity))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this case, $\\Delta_1$ in `d_in` is replaced with $\\Delta_2$.\n",
+    "\n",
+    "* $\\Delta_0$: how many values an individual may influence\n",
+    "* $\\Delta_2$: the euclidean influence an individual may have over all values\n",
+    "* $\\Delta_\\infty$: the influence an individual may have on any one value\n",
+    "\n",
+    "`m_lap` measures privacy with epsilon and delta (in the `Approximate<MaxDivergence>` measure), and `m_gauss` measures privacy with rho and delta (in the `Approximate<ZeroConcentratedDivergence>` measure).\n",
+    "\n",
+    "Notice how much smaller delta is this time (`2.8e-9` vs `1.1e-16`).\n",
+    "This is because the laplace distribution is a \"fat-tailed\" distribution, \n",
+    "meaning more of the mass of the distribution is in the tails.\n",
+    "The tails of the gaussian distribution decay much more quickly, \n",
+    "resulting in a much smaller delta.\n",
+    "\n",
+    "For comparison, lets convert the privacy guarantee from approx-zCDP to compare with the thresholded laplace mechanism:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "(ε, δ): (6.3035767282855915, 2.801398224505647e-09)\n"
+     ]
+    }
+   ],
+   "source": [
+    "# convert ρ to an ε(δ_2) privacy profile, where total privacy loss is (ε(δ_2), δ_1 + δ_2)\n",
+    "m_gauss_profile = dp.c.make_zCDP_to_approxDP(m_gauss)\n",
+    "# fix overall δ to that used by the Laplace threshold, for comparison\n",
+    "m_gauss_approx = dp.c.make_fix_delta(m_gauss_profile, delta=2.801398224505647e-09)\n",
+    "\n",
+    "print(\"(ε, δ):\", m_gauss_approx.map(sensitivity))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this setting, at the same level of delta as the thresholded laplace mechanism, \n",
+    "the privacy loss of the thresholded gaussian mechanism is over four times larger.\n",
+    "On the other hand, the thresholded gaussian mechanism will perform much better than the thresholded laplace mechanism\n",
+    "when $\\Delta_\\infty$ is small and $\\Delta_0$ is large.\n",
```

Comment:
I worry a bit about confusion coming from spelling "delta" out in some places and using the Greek in others. If they mean the same thing, use one consistently... but if they should be different, they probably aren't different enough? (See also the python code in adjacent cell.)

Reply:
I switched references in markdown to latex, to hopefully clear any ambiguity. The code still mentions delta, but only ever $\delta$, not $\Delta$.